storio-common/src/test/java/com/pushtorefresh/storio/internal/ChecksTest.java

```diff
@@ -34,9 +34,9 @@ public void checkNotNullNegative() {
     public void checkNotNullExceptionMessage() {
         try {
             checkNotNull(null, "expected message");
-            fail("NullPointerException should be thrown");
+            failBecauseExceptionWasNotThrown(NullPointerException.class);
```

Comment:
Let's use one style everywhere (in changes above you use message instead of exception class)